app/models/flatirons/saas/subscription.rb

```diff
@@ -48,5 +49,12 @@ def create_stripe_subscription
     def update_stripe_subscription
       stripe_service.update_subscription(stripe_subscription_id, stripe_price_id) if stripe_price_id_changed?
     end
+
+    def destroy_stripe_subscription
+      stripe_service.delete_subscription(stripe_subscription_id)
+    rescue StandardError => e
+      errors.add :base, :invalid, message: e.to_s
+      throw :abort
+    end
```

Comment:
Should we catch the error?
I guess any error will rollback the transaction, isn't it?

Reply:
if abort isnt thrown, transaction will not be rolled back